frontend/src/scenes/billing/BillingLimitInput.tsx

```diff
@@ -20,16 +20,27 @@ export const BillingLimitInput = ({ product }: { product: BillingProductV2Type }
     )
     const { setIsEditingBillingLimit, setBillingLimitInput } = useActions(billingProductLogic({ product }))
 
+    const [billinglimitError, setBillingLimitError] = useState('')
```

Comment:
this should actually probably be a value in the billingProductLogic .And this should actually be a kea form, which has built-in error handling 🙈 I'll let you decide if you want to do that refactor.